7.3.Частые_сценарии/7.3.8.py

```diff
@@ -0,0 +1,12 @@
+# Сумма чисел 2
+
+n = int(input())
+total = 0
+for i in range(1, n + 1):
+    if i ** 2 % 10 == 2:
+        total += i
+    elif i ** 2 % 10 == 5:
+        total += i
+    elif i ** 2 % 10 == 8:
+        total += i
```

Comment:
Можно так, а можно одной проверкой, у тебя же ИЛИ:
```suggestion
for i in range(1, n + 1):
    if i ** 2 % 10 == 2 or i ** 2 % 10 == 5 or i ** 2 % 10 == 8:
        total += i
```